docs/testing.md

```diff
@@ -1 +1,109 @@
-# Testing KBase UI code
\ No newline at end of file
+# Testing KBase UI code
```

Comment:
Bill, this testing document is excellent! I just had a few minor comments--see below.